packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.tsx

```diff
@@ -924,6 +935,150 @@ export class FocusZone extends React.Component<IFocusZoneProps> implements IFocu
 
     return false;
   }
+  private _moveFocusPaging(
+    isForward: boolean,
+    getDistanceFromCenter: (activeRect: ClientRect, targetRect: ClientRect) => number,
+    useDefaultWrap: boolean = true
+  ): boolean {
+    if (useDefaultWrap === void 0) {
+      useDefaultWrap = true;
+    }
+    let element = this._activeElement;
+    let candidateDistance = -1;
+    let candidateElement = undefined;
+    let changedFocus = false;
+    let pagesize = 0;
+    let targetTop = -1;
+    let targetBottom = -1;
+
+    if (!element || !this._root.current) {
+      return false;
+    }
+    if (this._isElementInput(element)) {
+      if (!this._shouldInputLoseFocus(element as HTMLInputElement, isForward)) {
+        return false;
+      }
+    }
+    const scrollableParent = findScrollableParent(element);
+    if (scrollableParent) {
```

Comment:
Consider changing this to `if (!scrollableParent) { return false; }` and then you could return true where you're setting `changedFocus = true` currently and the last `return changedFocus` could be `return false`. I'm suggesting this because currently that `if(scrollableParent)` check  encompasses most of the body of the function

Reply:
Also, it looks like most of the member variables you define at the beginning of this function could be scoped down because currently there's cases where you create those variables but they're never used